mpsl/CHANGELOG.rst

```diff
@@ -15,6 +15,9 @@ Changes
   using :cpp:func:`mpsl_timeslot_session_count_set`. All APIs timeslot APIs now take
   a session_id as input. The session id is retrieved from :cpp:func:`mpsl_timeslot_session_open()`.
 
+* Added API to use Front-End Modules (nRF21540 GPIO and simple GPIO) by the protocols and API to
+  configure them by the application. Only nRF52 series is supported.
```

Comment:
Maybe add " and 802.15.4"

Reply:
MPSL is actually protocol-agnostic. The thing that only 802.15.4 uses this FEM API should be mentioned somewhere else.

Reply:
We should in either case disallow enabling this feature when BLE is used

Reply:
Agree. That shall be handled by the Kconfigs in sdk-nrf/ repo.